.gitignore

```diff
@@ -37,4 +37,4 @@ jspm_packages
 .node_repl_history
 
 # Build directory
-dist
+#dist
```

Comment:
@HHogg I removed files from `dist` directory and added it back to `.gitignore`. I had trouble using the package through git url with YARN - for some reason it needed compiled version in dist folder.